aspnetcore/migration/22-to-30.md

```diff
@@ -1076,7 +1076,7 @@ new HubConnectionBuilder()
 
 ### Switch to Newtonsoft.Json
 
-If you're using features of `Newtonsoft.Json` that aren't supported in `System.Text.Json`, you can switch back to `Newtonsoft.Json`:
+If you're using [features of Newtonsoft.Json that aren't supported in System.Text.Json](/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to), you can switch back to `Newtonsoft.Json`:
```

Comment:
The instructions to switch back to `Newtonsoft.Json` seem to be spread across this doc depending on what type of app/technology folks are using (signalr vs mvc, etc). I wonder if this can be re-structured to be more clear under a single heading about switching with subheadings for the subtle differences, while highlighting what's common.

**Maybe, something like this:**

Heading: Switching the default JSON serializer back to `Newtonsoft.Json`
Intro: Aspnet uses S.T.Json by default, shipped in 3.0, etc.
To switch back you need to add the adapter package depending on the type of app you are building and add `AddNewtonsoftJson()` to your startup configure services or connection builder, etc.

Sub-heading: Aspnet Mvc project:
Install `<this relevant>` package
Change `<this>` code

Common errors/failures that can occur if you don't do this, or do it incorrectly.

Sub-heading: Aspnet Signalr project:
Install `<this relevant>` package
Change `<this>` code

Common errors/failures that can occur if you don't do this, or do it incorrectly.

Other sub-headings as needed:

@pranavkm - what do you think?

Reply:
Sure, that sounds great.

Reply:
I'll restructure the doc per Ahson's suggestion while applying the other feedback.

Reply:
This is ready for review.  A couple of notes

* The other h2 headings either state what was changed or are about how to adapt to what was changed -- this is the only one focused on how to undo what was changed. 
* I'm not aware of any other common errors/failures than the one highlighted in the issue @ahsonkhan mentioned.
* This structure consolidates the back-to-Newtonsoft content in one place, but now for SignalR the configure-S.T.Json content is separated from the back-to-Newtonsoft content.  I kept the back-to-Newtonsoft section heading in the SignalR section and linked back to the new location for the content. Another option is to duplicate the SignalR-back-to-Newtonsoft content to keep it in both places.